Modules/UpendoDnnSimpleAuthProvider/Login.ascx.cs

```diff
@@ -403,17 +401,16 @@ private void OnLoginClick(object sender, EventArgs e)
                             // The entered verification code is invalid (does not exist in the database).
                             // Set the login status to indicate a login failure and display an error message.
                             loginStatus = UserLoginStatus.LOGIN_FAILURE;
-                            EventLogController.Instance.AddLog("User Login - Invalid Verification Code", "Username: " + userName, PortalController.Instance.GetCurrentSettings(), objUser.UserID, EventLogController.EventLogType.LOGIN_FAILURE);
```

Comment:
@ArielBlanco1990 Why aren't you writing this event to the log anymore?

Reply:
@ArielBlanco1990  Okay.  Let's keep it in for now, but comment it out.  The same with the other one I asked about.